.github/actions/publish-logs/action.yml

```diff
@@ -48,6 +51,7 @@ runs:
       with:
         name: eden-report-tpm-${{ inputs.tpm_enabled }}-${{ inputs.file_system }}
         path: |
+            ${{ github.workspace }}/${{ inputs.info_file }}
```

Comment:
Maybe this can be inside `collect-info/action.yml`? Then we would not need to specify `info_file` for `publish-logs` action.

Reply:
Well, we can specify default value for input or hard-code it but if you change it you might lose it. But my intention was to be transparent that we save this file in some filepath and then we use this file in `publish-logs` .
I think I'm fine with hard-coding it here, since it's just one filepath and if you change it you can grep it.

Reply:
But this comment was about the placement of publishing of the info tarball. It would make more sense to publish it from collect-info/action.yml because it is unrelated to logs collecting.

Reply:
Or maybe we need to collect all artifacts from the same `actions/upload-artifact` action, I'm not sure.

Reply:
Well it's always a tradeoff :D. I think we can hard-code path and have one place to upload all artifacts